tensorly/decomposition/_symmetric_cp.py

```diff
@@ -33,30 +32,30 @@ def symmetric_power_iteration(tensor, n_repeat=10, n_iteration=10, verbose=False
     """
     order = tl.ndim(tensor)
     size = tl.shape(tensor)[0]
-    
+
     if not tl.shape(tensor) == (size, )*order:
         raise ValueError('The input tensor does not have the same size along each mode.')
 
     # A list of candidates for each mode
-    best_score = 0
+    best_score = -np.inf
```

Comment:
Best not to mix numpy in the calculations, let's just not initialize best_score here.